internal/compiler-interface/src/main/java/xsbti/compile/DefinesClass.java

```diff
@@ -8,14 +8,19 @@
 package xsbti.compile;
 
 /**
- * Determines if a classpath entry contains a class.
+ * Determine whether a classpath entry contains a class.
  *
- * The corresponding classpath entry is not exposed by this interface. It's tied
- * to it by a specific class that implements this interface.
+ * The corresponding classpath entry is not exposed by this interface.
  */
 public interface DefinesClass {
     /**
-     * Returns true if the classpath entry contains the requested class.
+     * Return true if the classpath entry contains the requested class.
```

Comment:
If you're happy to accept a Java style guide, there's this:

> + Use 3rd person (descriptive) not 2nd person (prescriptive). 
The description is in 3rd person declarative rather than 2nd person imperative.
>
> Gets the label. (preferred)
>
> Get the label. (avoid)

http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#styleguide